### PR TITLE
Fix misleading comment

### PR DESCRIPTION
**Pull request by jcpunk**
This fixes a misleading comment so that the documented behavior matches the code.

### CODE REVIEW

**Comment by poncovka** [comment]
Jenkins, test this please.

File: pyanaconda/payload/dnfpayload.py [hunk 1]
```diff
@@ -1141,8 +1141,8 @@ def update_base_repo(self, fallback=True, checkmount=True):
         # Enable or disable updates.
         self.set_updates_enabled(self._updates_enabled)
 
-        # Repos on disk are always enabled. When reloaded their state needs to
-        # be synchronized with the user selection.
+        # Repos from .treeinfo are always enabled.
+        # When reloaded their state needs to be synchronized with the user selection.
```
Comment (jkonecny12) [inline pyanaconda/payload/dnfpayload.py]: This is not correct. The `.treeinfo` is not taken into account here at all. They are loaded later, line 1159. This is only about  repo files loaded from the local system. I would suggest this:

```
Repo files are always loaded from the system.
When reloaded their state needs to be synchronized with the user configuration. So disable them now and enable them later if required.
```

Comment (jcpunk) [reply pyanaconda/payload/dnfpayload.py]: Just pushed a fix.

**Comment by jkonecny12** [comment]
Jenkins, it's ok to test.

**Pull request merged**